phpunit.xml

```diff
@@ -10,7 +10,10 @@
         stopOnFailure="false">
     <testsuites>
         <testsuite name="Application Test Suite">
-            <directory suffix="Test.php">./tests</directory>
+            <directory suffix="Test.php">./tests/phpunit</directory>
+        </testsuite>
+        <testsuite name="Controller Test">
+            <directory suffix="Test.php">./tests/phpunit/ControllerTests</directory>
         </testsuite>
```

Comment:
There should be a testSuite specifically for service tests if there is one for controllers

Reply:
👍